pkg/operator/controller/ingress/controller_test.go

```diff
@@ -1515,6 +1524,20 @@ func Test_IsProxyProtocolNeeded(t *testing.T) {
 			platform:    &awsPlatform,
 			expect:      false,
 		},
+		{
+			description: "loadbalancer strategy with NLB, but a service with ELB should use PROXY",
+			strategy:    &loadBalancerStrategyWithNLB,
+			platform:    &awsPlatform,
+			service:     &serviceWithELB,
+			expect:      true,
+		},
+		{
+			description: "loadbalancer strategy with ELB, but a service with NLB shouldn't use PROXY",
+			strategy:    &loadBalancerStrategyWithNLB,
```

Comment:
```suggestion
			strategy:    &loadBalancerStrategy,
```
based on the description of the test case.